__tests__/app.test.js

```diff
@@ -100,10 +98,89 @@ describe("GET /api/articles", () => {
 
         });
     });
+    test("should be sorted by article_id in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=article_id")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+
+          expect(articles).toBeSortedBy("article_id", { descending: true, coerce: true });
+
+        });
+    });
+    test("should be sorted by title in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+
+          expect(articles).toBeSortedBy("title", { descending: true, coerce: true });
+
+        });
+    });
+    test("should be sorted by votes in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+
+          expect(articles).toBeSortedBy("votes", { descending: true, coerce: true });
+
+        });
+    });
+    test("should be sorted by author in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+
+          expect(articles).toBeSortedBy("author", { descending: true, coerce: true });
+
+        });
+    });
+    test("should be sorted by topic in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=topic")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+
+          expect(articles).toBeSortedBy("topic", { descending: true, coerce: true });
+
+        });
+    });
+    test("should be sorted by author in ascending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=author&order=asc")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+
+          expect(articles).toBeSortedBy("author", { ascending: true, coerce: true });
+
+        });
+    });
```

Comment:
I would prefer to see the test for just the query order first and then a combined test like this after